lib/node_modules/@stdlib/array/complex64/lib/main.js

```diff
@@ -1919,6 +1919,136 @@ setReadOnly( Complex64Array.prototype, 'set', function set( value ) {
 	/* eslint-enable no-underscore-dangle */
 });
 
+/**
+* Returns a new typed array containing the extracted elements.
+*
+* @name slice
+* @memberof Complex64Array.prototype
+* @type {Function}
+* @param {integer} [start=0] - starting index (inclusive)
+* @param {integer} [end] - ending index (exclusive)
+* @throws {TypeError} `this` must be a complex number array
+* @throws {TypeError} first argument must be an integer
+* @throws {TypeError} second argument must be an integer
+* @returns {Complex64Array} complex number array
+*
+* @example
+* var realf = require( '@stdlib/complex/realf' );
+* var imagf = require( '@stdlib/complex/imagf' );
+*
+* var arr = new Complex64Array( 5 );
+*
+* arr.set( [ 1.0, -1.0 ], 0 );
+* arr.set( [ 2.0, -2.0 ], 1 );
+* arr.set( [ 3.0, -3.0 ], 2 );
+* arr.set( [ 4.0, -4.0 ], 3 );
+* arr.set( [ 5.0, -5.0 ], 4 );
+*
+* var slicearr = arr.subarray();
+* // throws {}
+*
+* var len = slicearr.length;
+* // returns 5
+*
+* var z = slicearr.get( 0 );
+* // returns <Complex64>
+*
+* var re = realf( z );
+* // returns 1.0
+*
+* var im = imagf( z );
+* // returns -1.0
+*
+* z = slicearr.get( len-1 );
+* // returns <Complex64>
+*
+* re = realf( z );
+* // returns 5.0
+*
+* im = imagf( z );
+* // returns -5.0
+*
+* subarr = arr.slicearray( 1, -2 );
+* // throws <TypeError>
+*
+* len = slicearr.length;
+* // returns 5
+*
+* z = slicearr.get( 0 );
+* // returns <Complex64>
+*
+* re = realf( z );
+* // returns 1
+*
+* im = imagf( z );
+* // returns -1
+*
+* z = slicearr.get( len-1 );
+* // returns <Complex64>
```

Comment:
```suggestion
* var out = arr.slice();
* // returns <Complex64Array>
*
* var len = out.length;
* // returns 5
*
* var z = out.get( 0 );
* // returns <Complex64>
*
* var re = realf( z );
* // returns 1.0
*
* var im = imagf( z );
* // returns -1.0
*
* z = out.get( len-1 );
* // returns <Complex64>
*
* re = realf( z );
* // returns 5.0
*
* im = imagf( z );
* // returns -5.0
*
* out = arr.slice( 1, -2 );
* // returns <Complex64Array>
*
* len = out.length;
* // returns 5
*
* z = out.get( 0 );
* // returns <Complex64>
*
* re = realf( z );
* // returns 1
*
* im = imagf( z );
* // returns -1
*
* z = out.get( len-1 );
* // returns <Complex64>
```

@Planeshifter This is another instance of ESLint auto-fixing. It would be really great to get this fixed once and for all, as this is a consistent headache and source of PR noise.